src/app.ts

```diff
@@ -32,15 +32,6 @@ import { ALL_MODULES } from './modules.js'
 import { integrationConfigPlugin } from './plugins/integrationConfigPlugin.ts'
 
 const GRACEFUL_SHUTDOWN_TIMEOUT_IN_MSECS = 10000
-const API_VERSION = '2.1.1'
-
-const getMajorApiVersion = (): string => {
```

Comment:
this should be handled on a contract per-endpoint level instead, having global api version is useless